### PR TITLE
Add Slack + Github actions and fix CODEOWNERS file

### PR DESCRIPTION
**Pull request by kideh88**
## Rationale

We (maintainers) should be aware of any PRs that are waiting for reviews

## Changes

- Add workflow for sending slack message when 'Ready for review' label has been added
- Add workflow for reminding about pending PR reviews every 3 hrs (AFAIK will not be active until it is merged to main)
- Remove label once PR has been approved
- Update the CODEOWNERS file

### CODE REVIEW

**Comment by kideh88** [comment]
Note for closing/reopening: Testing Slack message integration

File: .github/workflows/notify-slack.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            const labels = pullRequest.labels.map(label => label.name);
+
+            if (labels.includes(labelToRemove)) {
+              await github.rest.issues.removeLabel({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                name: labelToRemove
+              });
+              console.log(`Removed label: ${labelToRemove}`);
```
Comment (raulpineda) [inline .github/workflows/notify-slack.yml:64]: Are these console.log useful?

**Review by raulpineda — DISMISSED** [review]

File: .github/workflows/notify-slack.yml (hunk 1 above)
Comment (kideh88) [reply .github/workflows/notify-slack.yml:64]: Can you help me check it by approving the PR :smile: ? I havent been able to confirm this workflow action yet (cant approve my own PR) The console log should appear in the repos Actions tab but not 100% sure yet

Comment (kideh88) [reply .github/workflows/notify-slack.yml:64]: hmm this action didnt work, will need to revisit it :frowning_face:

**Comment by raulpineda** [comment]
@kideh88 it seems to have worked 🥳

File: .github/workflows/notify-slack.yml (hunk 1 above)
Comment (kideh88) [reply .github/workflows/notify-slack.yml]: confirmed that logs appear in the Actions tab :+1: : 
image

**Pull request merged**